### ISSUE TITLE
crash when trying to commit

### ISSUE THREAD

**Issue by mramato** [kentcdodds/validate-commit-msg#16]
Originally discussed in https://github.com/karma-runner/karma/pull/1816 but they suggested I open an issue here.

Whenever I try to commit to the karma repository (or rather my fork of it), validate-commit-msg crashes with the following error:

```
C:\Git\mramato\karma\node_modules\validate-commit-msg\index.js:103
      return buffer.toString().split('\n').shift();
                   ^

TypeError: Cannot read property 'toString' of undefined
    at firstLineFromBuffer (C:\Git\mramato\karma\node_modules\validate-commit-msg\index.js:103:20)
    at ReadFileContext.callback (C:\Git\mramato\karma\node_modules\validate-commit-msg\index.js:92:15)
    at FSReqWrap.readFileAfterOpen [as oncomplete] (fs.js:303:13)
```

This happens when the commit message is valid.

Here are my system specs:
Windows 7 
Node 4.2.x LTS
Git version 2.6.2.windows.1

Let me know if there's any other information you need or anything I can do to help track down the issue.  Thanks.

**Comment by kentcdodds** [comment]
I need to give you an apology. For some reason, I want watching this repository so I want getting notifications about issues and pull requests. I am so incredibly sorry. Please forgive me. I really feel awful. I hope my unresponsiveness doesn't hinder you contributing more in the future.

As for your specific pull request, I'll give it a closer look soon.

**Comment by kentcdodds** [comment]
This should be fixed by #10 Let me know if it's not.

**Comment by mramato** [comment]
Awesome, thanks.  There's no need to apologize, I know maintaining an OSS project is anything but easy.
